regression/contracts/quantifiers-forall-both-02/main.c

```diff
@@ -0,0 +1,18 @@
+// clang-format off
+int f1(int *arr, int *len) __CPROVER_requires(__CPROVER_forall {
+  int i;
+  (0 <= i && i < len) ==> arr[i] == 0
```

Comment:
This looks incorrect.

Did you mean `int len` in place of `int *len` in the function signature?